lib/supply/uploader.rb

```diff
@@ -24,11 +24,23 @@ def perform_upload
 
       upload_binary unless Supply.config[:skip_upload_apk]
 
+      promote_track
+
       Helper.log.info "Uploading all changes to Google Play..."
       client.commit_current_edit!
       Helper.log.info "Successfully finished the upload to Google Play".green
     end
 
+    def promote_track
+      if Supply.config[:track_promote_to]
+        version_codes = client.track_version_codes(Supply.config[:track])
+        client.update_track(Supply.config[:track], 1.0, nil)
```

Comment:
maybe we want to extract the following code
https://github.com/fastlane/supply/blob/0.4.0/lib/supply/uploader.rb#L93-L96
into one method and reuse it here (to get logging and rollout parameter support)